### PR TITLE
Fixed JWT signature calculation (1.8.x release)

### PR DESCRIPTION
**Pull request by hvge**
This PR fixes JWT signature calculation for 1.8.x release branch

### CODE REVIEW

File: proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/sdk/PowerAuthSDK.java [hunk 1]
```diff
@@ -2530,22 +2545,16 @@ public void onBiometricDialogFailed(@NonNull PowerAuthErrorException error) {
     @Nullable
     public ICancelable signJwtWithDevicePrivateKey(@NonNull Context context, @NonNull PowerAuthAuthentication authentication, @NonNull Map<String, Object> claims, @NonNull IJwtSignatureListener listener) {
         final JsonSerialization serialization = new JsonSerialization();
-        final byte[] serializedClaims = serialization.serializeObject(claims);
-        return signDataWithDevicePrivateKey(context, authentication, serializedClaims, new IDataSignatureListener() {
+        final String jwtHeader = "eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9"; // {"alg":"ES256","typ":"JWT"}
```
Comment (petrdvorak) [inline proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/sdk/PowerAuthSDK.java:2548]: This feels a bit arbitrary, could we extract it to some constant, i.e., `JwtHeader.ES256`?

File: proj-xcode/PowerAuth2/PowerAuthSDK.m [hunk 2]
```diff
@@ -1487,35 +1489,41 @@ - (NSData*) generateInvalidBiometricKey
...
+                                       format:PowerAuthCoreSignatureFormat_ECDSA_DER
+                                     callback:callback];
+}
+
 - (id<PowerAuthOperationTask>) signJwtWithDevicePrivateKey:(PowerAuthAuthentication*)authentication
                                                     claims:(NSDictionary<NSString*, NSObject*>*)claims
                                                   callback:(void(^)(NSString *jwt, NSError *error))callback
 {
+    // Prepare JWT Header
+    NSString * jwtHeader = @"eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9."; // {"alg":"ES256","typ":"JWT"}
```
Comment (petrdvorak) [inline proj-xcode/PowerAuth2/PowerAuthSDK.m:1507]: dtto, extraction to some constant would probably be helpful.

**Review by petrdvorak — APPROVED** [review]
Looks OK, I have two comments on style (using inline constant feels a bit arbitrary).

File: proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/sdk/PowerAuthSDK.java (hunk 1 above)
Comment (hvge) [reply proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/sdk/PowerAuthSDK.java:2548]: Objective-C is already enough ugly language, so adding 4-5 more lines just to be "nice" makes no sense to me :)

**Review by kober32 — APPROVED** [review]
Looks OK

**Pull request merged**